File: exercises/beecrowd/1004/1004.ts
// https://judge.beecrowd.com/pt/problems/view/1004
import * as fs from 'fs';

const input = fs.readFileSync('/dev/stdin', 'utf8');
const lines = input.split(/\r?\n/);

const a = Number(lines.shift());
const b = Number(lines.shift());

console.log(`PROD = ${a * b}`);

File: exercises/beecrowd/1004/1004.go
// https://judge.beecrowd.com/pt/problems/view/1004
package main

import (
	"fmt"
)

func main() {
	var a int
	var b int

	fmt.Scanln(&a)
	fmt.Scanln(&b)

	fmt.Println("PROD =", (a * b))
}
